### ISSUE TITLE
python-magic-bin

### ISSUE THREAD

**Issue by n0mad-samurai** [flamusdiu/xleapp#51]
I cannot seem to solve this issue::

The conflict is caused by:
    xleapp 0.2.1 depends on python-magic-bin<0.5.0 and >=0.4.14
    xleapp 0.2.0 depends on python-magic-bin<0.5.0 and >=0.4.14
    xleapp 0.1.1 depends on python-magic-bin<0.5.0 and >=0.4.14
    xleapp 0.1.0 depends on python-magic-bin<0.5.0 and >=0.4.14

Even though I have::

Name: python-magic-bin
Version: 0.4.14
Summary: File type identification using libmagic binary package
Home-page: http://github.com/julian-r/python-magic
Author: Adam Hupp, Julian David Rath, Spencer Hanson
Author-email: adam@hupp.org, julian.rath@gmail.com, spencer.hanson@swimlane.com
License: MIT
Location: /home/n0mad/.local/lib/python3.10/site-packages
Requires: 
Required-by:

**Comment by n0mad-samurai** [comment]
I downloaded the xLeapp 2.1 wheel package and removed the python-magic-bin requirement from the 'METADATA' file. The python-magic-bin is probably not needed in Linux so I will try without it. The xLeapp wheel package installed. Now I have a Jinja2 error when executing xleapp.

**Comment by flamusdiu** [comment]
Yeah, seems there is an issue with that package (https://github.com/julian-r/python-magic). At the moment, I moved to using the https://pypi.org/project/python-magic. I need to do more testing to see if that breaks anything on the main branch.

**Comment by flamusdiu** [comment]
I moved over to using the `python-magic` package. I need to test this configuration on different os'es. I fear it requires a second step to install `xleapp`.
